gnocchiclient/json.py

```diff
@@ -0,0 +1,58 @@
+# -*- encoding: utf-8 -*-
+#
+# Copyright © 2015-2017 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+import datetime
+import uuid
+
+import numpy
+import six
+import ujson
+
+
+def to_primitive(obj):
```

Comment:
my bad I have copied/pasted the gnocchi code, nothing is needed here the raw ujson is enough